### DISCUSSION TITLE
Automatically closing the drawer

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2805] [Q&A] [answered]
Hi, I'm integrating the landing page template and there's a little detail I'd like to polish. Would it be possible to have the drawer automatically close when I click on a section? As it goes to an anchor in the same page, the menu stays open after I click, which feels weird and makes me click in the page content in order to close it.

Also, I have to scroll up a little bit so I can see the title of the section. It would be nice to have the anchor a little above so I don't need to scroll.

See this video:

https://github.com/saadeghi/daisyui/assets/2629/0d9e2f46-6388-4271-a9f0-cd557670ff93

**Comment by saadeghi** [comment]
Thanks for reporting. I will fix it soon.

**Comment by saadeghi** [comment, marked as answer]
Fixed in version 1.1.0  
The file is updated and you can download the new version https://app.lemonsqueezy.com/my-orders
